<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PFindDialog</class>
 <widget class="QDialog" name="PFindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Text - musredit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../musredit.qrc">
    <normaloff>:/images/musrfit.xpm</normaloff>:/images/musrfit.xpm</iconset>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>342</width>
     <height>264</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="fFind_groupBox">
      <property name="title">
       <string>Find</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="fFind_label">
           <property name="text">
            <string>Text to find:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="fFind_comboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="fOptions_groupBox">
      <property name="title">
       <string>Options</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="fCaseSensitive_checkBox">
           <property name="text">
            <string>C&amp;ase Sensitive</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="fFindBackwards_checkBox">
           <property name="text">
            <string>Find &amp;backwards</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="fWholeWordsOnly_checkBox">
           <property name="text">
            <string>&amp;Whole words only</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="fSelectedText_checkBox">
           <property name="text">
            <string>&amp;Selected text</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="fFromCursor_checkBox">
           <property name="text">
            <string>From c&amp;ursor</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>168</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fFind_pushButton">
        <property name="text">
         <string>&amp;Find</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fClose_pushButton">
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>fFind_comboBox</tabstop>
  <tabstop>fCaseSensitive_checkBox</tabstop>
  <tabstop>fWholeWordsOnly_checkBox</tabstop>
  <tabstop>fFromCursor_checkBox</tabstop>
  <tabstop>fFindBackwards_checkBox</tabstop>
  <tabstop>fSelectedText_checkBox</tabstop>
  <tabstop>fFind_pushButton</tabstop>
  <tabstop>fClose_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../musredit.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fClose_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PFindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fFind_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>PFindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fFind_comboBox</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>PFindDialog</receiver>
   <slot>onFindTextAvailable(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onFindTextAvailable(QString)</slot>
 </slots>
</ui>
